src/css/vote-CSS.css

```diff
@@ -0,0 +1,109 @@
+html {
+  display: table;
+}
+
+body {
+  padding: 100px;
+  background: linear-gradient(to right, lightblue, lightblue 50%, darkblue 50%);
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
+
+.test_box {
+  max-width: 100%;
+  height: auto;
+  padding: 20px;
+  border-radius: 25px;
+  background: linear-gradient(to left, lightblue, lightblue 50%, white 50%);
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
+
+.u-layout {
```

Comment:
So this is your most used class, but I think we can refactor this.

I think you need the width setting on `.test_box` (this needs renaming), as then all your child elements will be restricted to that width because it is the parent.

When the `width` is moved, the only thing this class is doing is adding a margin bottom and align text center. These can be two separate utility classes that can be used specifically; `u-text-center` and `u-mb-40`